jira/client.py

```diff
@@ -3090,13 +3076,11 @@ def search_assignable_users_for_projects(
         Args:
             username (str): A string to match usernames against
             projectKeys (str): Comma-separated list of project keys to check for issue assignment permissions
-            startAt (int): Index of the first user to return (Default: 0)
-            maxResults (int): Maximum number of users to return.
-              If maxResults evaluates as False, it will try to get all users in batches. (Default: 50)
+            startAt (int): Index of the first user to return (Default: ``0``)
+            maxResults (int): Maximum number of users to return. If maxResults evaluates as False, it will try to get all users in batches. (Default: ``50``)
```

Comment:
here is another example which follows my previous comment of summary and detail lines.
It also reaches 157 characters wide, which feels a bit long even for me.

But in this case I don't mind as these docstrings are mainly going to be read in the html sphinx docs.